src/back/spv/layout.rs

```diff
@@ -135,10 +135,8 @@ impl Instruction {
             inst_index += 1;
         }
 
-        let mut op_index = 0;
-        for i in inst_index..wc as usize {
+        for (op_index, i) in (inst_index..wc as usize).enumerate() {
```

Comment:
might as well do `for (operand, word) in self.operands.iter().zip(words[inst_index..wc as usize].iter()) { .. }`

Reply:
I just tried that however it gives an error about indexing with a &i32. 😔